opm/parser/eclipse/CMakeLists.txt

```diff
@@ -276,18 +286,13 @@ Utility/Functional.hpp
 Utility/Stringview.hpp
 Utility/Typetools.hpp)
 
-add_library(buildParser ${rawdeck_source} ${build_parser_source} ${deck_source} ${unit_source} ${generator_source} ${utility_source})
-target_link_libraries(buildParser opmjson ${opm-common_LIBRARIES} ${Boost_LIBRARIES}  ${ERT_LIBRARIES})
-
 #-----------------------------------------------------------------
 # This section manages the generation of C++ code for the default keywords.
 
 # 1. Create an executable 'createDefaultKeywordList'.
 
-add_executable( createDefaultKeywordList Parser/createDefaultKeywordList.cpp )
-target_link_libraries( createDefaultKeywordList buildParser opmjson ${Boost_LIBRARIES})
-
-
+add_executable(createDefaultKeywordList Parser/createDefaultKeywordList.cpp ${build_parser_source})
+target_link_libraries( createDefaultKeywordList opmjson ${Boost_LIBRARIES} ${ERT_LIBRARIES})
```

Comment:
No, it still uses a `util_call` somewhere. I tried, was disappointed, re-added and honestly figure that I think it's possible to remove it.

The entire AOT compilation could use an overhaul, this is just the lowest hanging fruit.